### PR TITLE
add smooth-scroll v7.0.2 with git autoupdate, close #5161

### LINKED ISSUE #5161: [Request] smooth-scroll
**Library name:** smooth-scroll
**Git repository url:** https://github.com/cferdinandi/smooth-scroll
**npm package url(optional):** cferdinandi/smooth-scroll
**Official homepage:** https://github.com/cferdinandi/smooth-scroll
**Wanna say something? Leave message here:** I really thought in sending a pull request, but the README sounded confusing, with that "IMPORTANT" message at the beginning and later on an explanation of how to proceed. Didn't get it. The process itself sounded confusing. Can we still contribute via pull requests? There's 87 open pr's. Best regards to all of you and sorry if I'm being a little dumb.
**cc:** @cferdinandi

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by maruilian11**
@LeaYeh could u plz help me check this pr for #5161 , thx~

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Any updates?

**Comment by maruilian11** [comment]
@LeaYeh could u help me check again? thx.

**Comment by LeaYeh** [comment]
but the unit test was not passing, check it again :)

**Comment by maruilian11** [comment]
@LeaYeh  sorry, i'm busy recently, could u help me check again? thx.

**Comment by maruilian11** [comment]
@Amomo LeaYeh seems to be busy currently, could u help me check this PR? thank u very much:))

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine. Could you take a look? Thank you~

**Comment by PeterDaveHello** [comment]
Nope, wrong structure, you need to check the consistency of the directory/file structure and filemap in auto-update.

**Comment by PeterDaveHello** [comment]
And let's add some **keywords** in package.json.

**Comment by maruilian11** [comment]
@Amomo i have change the basepath field and add keywords, could u help me check again? thank u~

**Comment by Amomo** [comment]
@PeterDaveHello I think the modification is fine.
Could you take a look? Thank you~

**Comment by PeterDaveHello** [comment]
Why don't you add `js` folder for it? I think keep the structure as official will be a better way.

**Comment by maruilian11** [comment]
@PeterDaveHello ok. got it.

**Comment by maruilian11** [comment]
@Amomo sorry, let you guys check and check again, could u plz check it again? thank u very much !

**Comment by LeaYeh** [comment]
i think Peter means the filemap should be `js/*`

**Comment by maruilian11** [comment]
@Amomo @LeaYeh  i see !  i have corrected the filemap field, could u help me check again? thank you very much !

**Comment by LeaYeh** [comment]
i think it is fine. @PeterDaveHello

File: ajax/libs/smooth-scroll/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/cferdinandi/smooth-scroll"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/cferdinandi/smooth-scroll.git",
+    "basePath": "dist",
+    "files": [
+      "js/*"
```
Comment (PeterDaveHello) [inline ajax/libs/smooth-scroll/package.json]: Why don't you use ‵**/*`?

**Comment by maruilian11** [comment]
@LeaYeh i think we just add the subdirectories on local host to kill original structure, we still can use <code>***/**</code>. do u have any ideas? could u help me check this PR again? thank you~

**Comment by LeaYeh** [comment]
ok, got it. @PeterDaveHello

**Pull request merged**